pkg/e2etemplates/aws_host_peer_stack_template.go

```diff
@@ -0,0 +1,32 @@
+package e2etemplates
+
+const awsHostPeerStackTemplate = `
+AWSTemplateFormatVersion: 2010-09-09
+Description: Host Peer Stack with VPC peering and route tables for testing purposes
+Resources:
+  VPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.11.0.0/16
```

Comment:
FMI why is this hard coded though and how does it influence the e2e test?

Reply:
AIUI we just need to make sure this doesn't clash with guest clusters. They have 10.12.0.0/24 so it's fine.